forms-flow-web/src/components/Draft/DraftFilter.js

```diff
@@ -148,13 +152,15 @@ const DraftFilter = ({ setDisplayFilter, filterParams, setFilterParams }) => {
           <button
             className="btn btn-link text-dark me-1 "
             onClick={closeFilterModal}
+            data-testid="draft-filter-close-button"
           >
             {t("Cancel")}
           </button>
           <button
             className="btn btn-dark"
             onClick={applyFilters}
             disabled={!isSearchParamEntered}
+            data-testid="draft-filter-apply-filter-button"
```

Comment:
"draft-filter-apply-button"  is enough , right?

Reply:
done